### DISCUSSION TITLE
are there any more cool or useful chrome urls like chrome://notification-tester/?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1376] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
list pls

**Comment by S-PScripts** [comment]
chrome:about just check them out

**Reply by S-PScripts** [reply]
nothing too interesting tbh

**Reply by ghost** [reply]
Thanks but chrome://about is blocked for me.

**Reply by S-PScripts** [reply]
chrome:chrome-urls

**Reply by S-PScripts** [reply]
they're identical idk why google has chrome://about and chrome://chrome-urls

**Comment by S-PScripts** [comment]
chrome://about/
chrome://access-code-cast
chrome://accessibility/
chrome://accessory-update/
chrome://account-manager-error/
chrome://account-migration-welcome/
chrome://add-supervision/
chrome://app-disabled/
chrome://app-install-dialog/
chrome://app-service-internals/
chrome://arc-overview-tracing/
chrome://arc-power-control/
chrome://assistant-optin/
chrome://attribution-internals/
chrome://audio
chrome://autofill-internals/
chrome://blob-internals/
chrome://bluetooth-internals/
chrome://bluetooth-pairing/
chrome://bookmarks/
chrome://bookmarks-side-panel.top-chrome/
chrome://borealis-credits/
chrome://borealis-installer/
chrome://camera-app/
chrome://cast-feedback/
chrome://certificate-manager/
chrome://chrome/
chrome://chrome-signin/
chrome://chrome-urls/
chrome://cloud-upload/
chrome://color-internals/
chrome://commerce-internals/
chrome://compare/
chrome://components/
chrome://confirm-password-change
chrome://connection-help/
chrome://connection-monitoring-detected/
chrome://connectivity-diagnostics/
chrome://connectors-internals/
chrome://crashes/
chrome://credits/
chrome://crostini-credits/
chrome://crostini-installer/
chrome://crostini-upgrader/
chrome://cryptohome/
chrome://customize-chrome-side-panel.top-chrome/
chrome://data-sharing-internals/
chrome://device-log/
chrome://diagnostics/
chrome://dino/
chrome://discards/
chrome://dlp-internals/
chrome://download-internals/
chrome://downloads/
chrome://drive-internals/
chrome://eche-app/
chrome://emoji-picker/
chrome://enterprise-reporting
chrome://extended-updates-dialog
chrome://extensions/
chrome://extensions-internals/
chrome://family-link-user-internals/
chrome://feedback/
chrome://file-manager/
chrome://files-internals/
chrome://flags/
chrome://focus-mode-media/
chrome://gcm-internals/
chrome://gpu/
chrome://graduation
chrome://growth-internals
chrome://healthd-internals/
chrome://help/
chrome://help-app/
chrome://histograms/
chrome://history/
chrome://history-clusters-internals/
chrome://history-clusters-side-panel.top-chrome/
chrome://identity-internals/
chrome://indexeddb-internals/
chrome://inspect/
chrome://internals/
chrome://internet-config-dialog/
chrome://internet-detail-dialog/
chrome://interstitials/
chrome://kerberos-in-browser/
chrome://kiosk-vision-internals
chrome://launcher-internals/
chrome://lens-overlay/
chrome://lens-search-bubble/
chrome://linux-proxy-config/
chrome://local-files-migration
chrome://local-state/
chrome://location-internals/
chrome://lock-network/
chrome://lock-reauth
chrome://mall
chrome://manage-mirrorsync
chrome://management/
chrome://media-app/
chrome://media-engagement/
chrome://media-internals/
chrome://media-router-internals/
chrome://memory-internals/
chrome://metrics-internals/
chrome://mobilesetup/
chrome://multidevice-internals/
chrome://multidevice-setup/
chrome://nacl/
chrome://nearby/
chrome://nearby-internals/
chrome://net-export/
chrome://net-internals/
chrome://network/
chrome://network-errors/
chrome://new-tab-page/
chrome://new-tab-page-third-party/
chrome://newtab/
chrome://notification-tester/
chrome://ntp-tiles-internals/
chrome://office-fallback/
chrome://omnibox/
chrome://omnibox-popup.top-chrome
chrome://on-device-internals/
chrome://oobe/
chrome://optimization-guide-internals/
chrome://os-credits/
chrome://os-feedback/
chrome://os-print
chrome://os-settings/
chrome://parent-access/
chrome://password-change
chrome://password-manager/
chrome://password-manager-internals/
chrome://personalization/
chrome://policy/
chrome://power/
chrome://predictors/
chrome://prefs-internals/
chrome://print/
chrome://print-management/
chrome://privacy-sandbox-dialog/
chrome://privacy-sandbox-internals
chrome://private-aggregation-internals/
chrome://process-internals/
chrome://profile-internals/
chrome://proximity-auth
chrome://quota-internals/
chrome://read-later.top-chrome/
chrome://recorder-app
chrome://reset-password/
chrome://safe-browsing/
chrome://sandbox/
chrome://sanitize
chrome://scanning/
chrome://search-engine-choice
chrome://security-curtain/
chrome://segmentation-internals/
chrome://sensor-info/
chrome://serviceworker-internals/
chrome://set-time/
chrome://settings/
chrome://shimless-rma
chrome://shopping-insights-side-panel.top-chrome/
chrome://shortcut-customization/
chrome://signin-internals/
chrome://site-engagement/
chrome://slow/
chrome://slow_trace/
chrome://smb-credentials-dialog/
chrome://smb-share-dialog/
chrome://suggest-internals/
chrome://support-tool/
chrome://sync-internals/
chrome://sys-internals/
chrome://system/
chrome://tab-search.top-chrome/
chrome://tab-strip.top-chrome/
chrome://terms/
chrome://topics-internals/
chrome://traces-internals/
chrome://tracing/
chrome://translate-internals/
chrome://ukm/
chrome://urgent-password-expiry-notification
chrome://usb-internals/
chrome://user-actions/
chrome://vc-background
chrome://vc-tray-tester/
chrome://version/
chrome://view-cert/
chrome://vm/
chrome://web-app-internals/
chrome://webrtc-internals/
chrome://webrtc-logs/
chrome://webui-gallery/
chrome://webuijserror/
chrome-untrusted://boca-app
chrome-untrusted://camera-app/
chrome-untrusted://companion-side-panel.top-chrome
chrome-untrusted://compose
chrome-untrusted://crosh/
chrome-untrusted://data-sharing
chrome-untrusted://demo-mode-app
chrome-untrusted://eche-app/
chrome-untrusted://file-manager/
chrome-untrusted://focus-mode-player/
chrome-untrusted://hats
chrome-untrusted://help-app/
chrome-untrusted://help-app-kids-magazine/
chrome-untrusted://lens/
chrome-untrusted://lens-overlay/
chrome-untrusted://mako
chrome-untrusted://media-app/
chrome-untrusted://os-feedback/
chrome-untrusted://print/
chrome-untrusted://privacy-sandbox-dialog/
chrome-untrusted://projector/
chrome-untrusted://projector-annotator/
chrome-untrusted://read-anything-side-panel.top-chrome/
chrome-untrusted://scalable-iph-debug
chrome-untrusted://terminal/
List of chrome://internals pages
chrome://internals/session-service
For Debug
The following pages are for debugging purposes only. Because they crash or hang the renderer, they're not linked directly; you can type them into the address bar if you need them.

chrome://badcastcrash/
chrome://inducebrowsercrashforrealz/
chrome://inducebrowserdcheckforrealz/
chrome://crash/
chrome://crash/rust
chrome://crashdump/
chrome://kill/
chrome://hang/
chrome://shorthang/
chrome://gpuclean/
chrome://gpucrash/
chrome://gpuhang/
chrome://memory-exhaust/
chrome://memory-pressure-critical/
chrome://memory-pressure-moderate/
chrome://webuijserror/
chrome://quit/
chrome://restart/

**Reply by S-PScripts** [reply]
@Ryler4 if chrome://chrome-urls is blocked too look at this list

**Reply by ghost** [reply]
Thanks so much! @S-PScripts
